pcdsdevices/epics_motor.py

```diff
@@ -174,6 +220,14 @@ def check_value(self, value):
             raise Exception("Motor is not enabled. Motion requests "
                             "ignored")
 
+        if self.motor_spg.value in [0, 'Stop']:
+            raise Exception("Motor is stopped.  Motion requests "
+                            "ignored until motor is set to 'Go'")
+
+        if self.motor_spg.value in [1, 'Pause']:
+            raise Exception("Motor is paused.  Motion requests "
+                            "ignored until motor is set to 'Go'")
+
```

Comment:
This is properly implemented just thinking out loud. It might be getting close to time to have a custom `Exception` for this kind of "Motor Configuration" Error. We now check quite a few things at the start of a move, and more descriptive errors are helpful for operators.

Reply:
I had the same thought